src/views/v2/month/calendar-body/day/multiday-events/multiday-event.php

```diff
@@ -93,7 +93,7 @@ class="tribe-events-calendar-month__multiday-event-hidden-featured-icon tribe-co
 					></em>
 				<?php endif; ?>
 				<h3 class="tribe-events-calendar-month__multiday-event-hidden-title tribe-common-h8">
-					<?php echo wp_kses_post( get_the_title( $event->ID ) ); ?>
+					<?php echo $event->title; ?>
```

Comment:
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$event'
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$event'.

<sub>hash: ec0c79ace7c6dffb57e28dfee7ea2654</sub>